internal/compiler/widgets/common/menu-base.slint

```diff
@@ -80,13 +81,12 @@ export component MenuBarBase inherits Rectangle {
 export component MenuFrameBase inherits Rectangle {
     in property <length> spacing <=> layout.spacing;
     in property <length> layout-min-width;
+    in property <length> margin <=> layout.padding;
```

Comment:
Ooops, this line didn't work.
The lower_layout pass don't see this two way binding and doesn't set the padding. That's yet another layouting bug.

Anyway, fixed now.